tests/meta/archive/test_archiving.py

```diff
@@ -305,25 +310,20 @@ def test_archive_folders(
 
 def test_retrieve_folders(
     ddn_dataflow_api: DDNDataFlowApi,
-    local_directory: Path,
-    remote_path: Path,
     full_remote_path: str,
     full_local_path: str,
-    ok_response: Response,
+    ok_ddn_response: Response,
+    transfer_data_retrieve: TransferData,
 ):
     """Tests that the retrieve function correctly formats the input and sends API request."""
 
     # GIVEN two paths that should be retrieved
 
     # WHEN running the retrieve method and providing two paths
     with mock.patch.object(
-        APIRequest,
-        "api_request_from_content",
-        return_value=ok_response,
+        APIRequest, "api_request_from_content", return_value=ok_ddn_response
     ) as mock_request_submitter:
-        response: bool = ddn_dataflow_api.retrieve_folders(
-            {Path(remote_path): Path(local_directory)}
-        )
+        response: int = ddn_dataflow_api.retrieve_folders([transfer_data_retrieve])
 
     # THEN the response returned should be true
     assert response
```

Comment:
```suggestion
    # THEN the response returned should be the job id
    assert response = ok_ddn_response{"job_id"}
```
similar to above I suppose.

Reply:
Fixed assert and comment